File: lib/py/uapi/internal/binary/PackBody.py
from typing import Any, Dict
from uapi.internal.binary.Pack import pack


def pack_body(body: Dict[Any, Any]) -> Dict[Any, Any]:
    result = {}

    for key, value in body.items():
        packed_value = pack(value)
        result[key] = packed_value

    return result

File: lib/ts/uapi/internal/binary/PackBody.ts
import { pack } from './Pack';

export function packBody(body: Record<string, any>): Record<string, any> {
    const result: Record<string, any> = {};

    for (const [key, value] of Object.entries(body)) {
        const packedValue = pack(value);
        result[key] = packedValue;
    }

    return result;
}
